style.css

```diff
@@ -107,11 +107,12 @@ h2, .checkUserGuess, #inputUsername {
   background-color: #55608E;
 }
 
```

Comment:
I commented out this button style because it is not needed; 'background-color' is supposed to be set dynamically from a click event (which is what this PR solves). 'color' is set earlier in the CSS so that's why it's not needed either.